enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/impl/service/IdentityVerificationService.java

```diff
@@ -212,16 +212,9 @@ public List<DocumentVerificationEntity> submitDocuments(DocumentSubmitRequest re
 
         final IdentityVerificationPhase phase = idVerification.getPhase();
         final IdentityVerificationStatus status = idVerification.getStatus();
-        if (phase == IdentityVerificationPhase.DOCUMENT_VERIFICATION && status == IdentityVerificationStatus.IN_PROGRESS) {
```

Comment:
This used to be a fix to allow uploading another document in different states. But we return to the previous behavior, to allow it only in `DOCUMENT_UPLOAD / IN_PROGESS`